src/components/DroughtHistory.vue

```diff
@@ -608,21 +813,71 @@ $writeFont: 'Nanum Pen Script', cursive;
   width: 0;
   height: 0;
 }
-#empty-div {
+.page-section {
+  margin: auto;
+  max-width: 90vw;
+}
+#region-grid-container {
+  max-width: 95vw;
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: max-content max-content;
+  grid-template-areas:
+    "map"
+    "wedges";
+}
+#casc-svg {
+  grid-area: map;
+  display: flex;
+  width: 100%;
+  height: 100%;
+}
+#wedges-svg {
+  grid-area: wedges;
+  display: flex;
+  width: 100%;
+  height: 100%;
+}
+#region-description {
+  height: 100px;
+}
+#methods-container {
+  height: 300px;
+}
+#references-container {
+  height: 300px;
+}
+#authors-container {
   height: 300px;
 }
 </style>
 <style lang="scss">
 .droughtText {
   z-index: 10;
-  font-weight: 300;
+  font-weight: 400;
   font-size: 1em;
+  -webkit-user-select: none; /* Safari */
+  -ms-user-select: none; /* IE 10 and IE 11 */
+  user-select: none; /* Standard syntax */
 }
 .droughtText.mobile {
   z-index: 10;
   font-weight: 500;
   font-size: 1em;
 }
+.droughtText.quote {
+  font-style: italic;
+}
+.droughtText.desktop.quote {
+  text-shadow: 2px 4px 4px rgba(179,179,179,0.6);
```

Comment:
wondering if a small neat line at bottom of the text box might help in this case, where it scrolls to next section? 

![image](https://github.com/DOI-USGS/drought-timeline/assets/19958841/41face98-7388-4a29-82d0-28239b5901eb)

Reply:
Yeah I think that does help:
![image](https://github.com/DOI-USGS/drought-timeline/assets/54007288/665baea1-4dd5-4e5a-a31e-809061916026)